guide/release-notes/v3.0.8.adoc

```diff
@@ -0,0 +1,19 @@
+= Version 3.0.8
+[horizontal]
+Release date:: 2020-11-01
+
+== Changes to BIS document
+* Definitions for EndPoint identifiers BT-34 and BT-49 aligned with defintion in EN16931. Issue: [POACC-331]
+* Migration plan from BIS2 to BIS3 removed. Is completed.
+* Use GLN as the only examlpe in section 10.2. Delivery Details (Date and Location). Removed reference to GSRN from the description text. [POACC-332]
+
+== Changes to support documents
+
+
+== Changes to code lists and validation artefacts
+* Rule PEPPOL-COMMON-R040: "GLN must have a valid format according to GS1 rules". Changed from severity "warning" to "fatal". (The rule was introduced in fall 2019 with severity "warning" to avoid disruptions but with the intention to be changed to severity "fatal" after 6-12 months). Issue: [POACC-333] and [POACC-341]
+* EAS code list, added code 0209, removed code 9958. Rule updated accordingly.
```

Comment:
Code lists in https://github.com/OpenPEPPOL/peppol-bis-invoice-3/tree/2020-Fall-Dev01/rules/sch do not appear to have this rule applied. (Both Peppol and EN)